integration-tests/utils/integration-utils.js

```diff
@@ -215,38 +215,10 @@ const dismissNPSSurvey = async page => {
 // Test workaround: Retry loading of Terra UI if fails first time. This issue often happens after new deploy to Staging/Alpha.
 const signIntoTerra = async (page, { token, testUrl }) => {
   console.log('signIntoTerra ...')
-
-  const retryOptions = {
-    factor: 1,
-    minTimeout: 1000, // This is min wait time between retries
-    onFailedAttempt: error => {
-      console.error(
-        `Sign in to Terra attempt ${error.attemptNumber} failed. There are ${error.retriesLeft} retries left.`
-      )
-    },
-    retries: 2
-  }
-
-  const timeout = 60 * 1000
-  const run = async url => {
-    try {
-      const httpResponse = await gotoPage(page, url)
-      if (!(httpResponse.ok() || httpResponse.status() === 304)) {
-        throw new Error(`Error loading URL: ${url}. Http response status: ${httpResponse.statusText()}`)
-      }
-      await page.waitForXPath('//*[contains(normalize-space(.),"Loading Terra")]', { hidden: true, timeout })
-    } catch (e) {
-      // Stop page loading, as if you hit "X" in the browser. ignore exception.
-      await page._client.send('Page.stopLoading').catch(err => void err)
-      throw new Error(e)
-    }
-  }
-
   if (!!testUrl) {
-    console.log(`Loading URL: ${testUrl}`)
-    await pRetry(() => run(testUrl), retryOptions)
+    await gotoPage(page, testUrl)
   } else {
-    await page.waitForXPath('//*[contains(normalize-space(.),"Loading Terra")]', { hidden: true, timeout })
+    await page.waitForXPath('//*[contains(normalize-space(.),"Loading Terra")]', { hidden: true, timeout: 60 * 1000 })
```

Comment:
591e92ca53f63dc1946b73eba41b641ce976758c removed the 60 second timeout from `gotoPage`. Should it also be removed here?

Reply:
Doesn't hurt to remove 60 sec wait. Done.